### PR TITLE
Fix NPE in shardingjdbc 1.5.4.1

### LINKED ISSUE #4765: Null pointer exception when using shardingjdbc 1.5.4.1
### Bug
Version of SkyWalking 7.0.0.When I install the skywalking agent on my project ,the following exception occurred.
4QTzcY

### PR DESCRIPTION
**Pull request by ynfeng**
For  #4765

### CODE REVIEW

File: apm-sniffer/apm-sdk-plugin/sharding-jdbc-1.5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/sjdbc/ExecuteEventListener.java [hunk 1]
```diff
@@ -62,13 +62,17 @@ private void handle(AbstractExecutionEvent event, String operation) {
...
                     ContextManager.continued((ContextSnapshot) ExecutorDataMap.getDataMap()
-                                                                              .get(AsyncExecuteInterceptor.SNAPSHOT_DATA_KEY));
+                        .get(AsyncExecuteInterceptor.SNAPSHOT_DATA_KEY));
                 }
                 Tags.DB_TYPE.set(span, "sql");
                 Tags.DB_INSTANCE.set(span, event.getDataSource());
                 Tags.DB_STATEMENT.set(span, event.getSql());
                 if (!event.getParameters().isEmpty()) {
-                    Tags.DB_BIND_VARIABLES.set(span, Joiner.on(",").join(event.getParameters()));
+                    String variables = event.getParameters()
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/sharding-jdbc-1.5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/sjdbc/ExecuteEventListener.java:71]: What thing is NULL triggering NPE?

Comment (ynfeng) [reply apm-sniffer/apm-sdk-plugin/sharding-jdbc-1.5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/sjdbc/ExecuteEventListener.java:71]: event.getParameters contains null value,inside Joiner will check null value and then throw NPE

Comment (ynfeng) [reply apm-sniffer/apm-sdk-plugin/sharding-jdbc-1.5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/sjdbc/ExecuteEventListener.java:71]: event.getParameters contains null value,inside Joiner will check null value and then throw NPE

Comment (ynfeng) [reply apm-sniffer/apm-sdk-plugin/sharding-jdbc-1.5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/sjdbc/ExecuteEventListener.java:71]: event.getParameters contains null value,inside Joiner will check null value and then throw NPE

Comment (wu-sheng) [reply apm-sniffer/apm-sdk-plugin/sharding-jdbc-1.5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/sjdbc/ExecuteEventListener.java:71]: Got it. LGTM.

**Pull request merged**